app/models/photo.rb

```diff
@@ -147,6 +147,10 @@ def update_default_license
   def update_all_licenses
     return true unless [true, "1", "true"].include?(@make_licenses_same)
     Photo.where(user_id: user_id).update_all(license: license)
+    Photo.where(user_id: user_id).pluck(:id).each do |photo_id|
```

Comment:
Have you confirmed this will run before a request times out if a person has like 100k photos?

Reply:
I haven't timed it, and I agree it's a liability. I heard we like delayed jobs, so I made a delayed job that will create these delayed jobs w/o affecting web response time

Reply:
Sing a sea shanties and down some Tide Pods and we'll be at web scale. Something to keep an eye on during testing, I guess.